src/main/java/seedu/teachbook/logic/commands/AddClassCommand.java

```diff
@@ -1,30 +1,25 @@
 package seedu.teachbook.logic.commands;
 
 import static java.util.Objects.requireNonNull;
-import static seedu.teachbook.logic.parser.CliSyntax.PREFIX_NAME;
 
 import seedu.teachbook.logic.commands.exceptions.CommandException;
 import seedu.teachbook.model.Model;
 import seedu.teachbook.model.classobject.Class;
 
-
 /**
- * Adds a student to the teachbook.
+ * Adds a student to TeachBook.
```

Comment:
Hmmm you wanna use `TeachBook`? We can discuss about this later...

Reply:
I can change it back first then haha